gon.hcl

```diff
@@ -1,9 +1,15 @@
 source = ["./konvoy-image-bundle_darwin_amd64/konvoy-image"]
 bundle_id = "com.d2iq.dkp.konvoy-image-wrapper"
+
 apple_id {
   username = "@env:AC_USERNAME"
   password = "@env:AC_PASSWORD"
 }
+
 sign {
   application_identity = "Developer ID Application: Mesosphere Inc. (JQJDUUPXFN)"
 }
+
+zip {
+  "output_path": "konvoy-image.zip"
+}
```

Comment:
Why do we need this zip?

Reply:
It is needed because it is what apple requires of things that can be notarized 

from gon docs 

> notarize (optional) - Settings for notarizing already built files. This is an alternative to using the source option. This option can be repeated to notarize multiple files.

>    path (string) - The path to the file to notarize. This must be one of Apple's supported file types for notarization: dmg, pkg, app, or zip.

Reply:
> zip (optional) - Settings related to creating a zip archive as output. A zip archive will only be created if this is specified. Note that zip archives don't support stapling, meaning that files within the notarized zip archive will require an internet connection to verify on first use.

this wont work for the airgap mac user though